frontend/src/metabase/query_builder/components/view/ExecutionTime/ExecutionTime.tsx

```diff
@@ -0,0 +1,15 @@
+import { t } from "ttag";
+
+import { Tooltip } from "metabase/ui";
+
```

Comment:
nit: I believe there shouldn't be an empty line between those groups as both they both point out to the relative files

Reply:
If it starts with a dot it's _relative_, if it's absolute it's _internal_, so all is good here

Reply:
that's not true, you know, it's a relative path with webpack paths mapping

Reply:
I'm not sure about terminology here but this is what `import/order` enforces with the current config